### PR TITLE
Bump all packages to 13.1.0

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

- Bumped SRP packages to 13.1.0
- Moved some release notes to the new section, as they were added after the commit used for the current vendoring (and therefore correspond to the next vendoring).

---
### Testing status
Package Tests (upm validation): https://unity-ci.cds.internal.unity3d.com/project/902/branch/srp%2Fbump-packages/jobDefinition/.yamato%2F_projectcontext.yml%23all_package_ci_project_trunk

### CODE REVIEW

**Review by sebastienlagarde — COMMENTED** [review]
see the various remark for 21.2 PR

**Review by marctem — APPROVED** [review]

**Comment by theo-pnv** [comment]
- Added the PR review requested changes
- Merged latest master
- Unbumped the templates as they will be bumped in a separate PR https://github.com/Unity-Technologies/Graphics/pull/5799

File: com.unity.render-pipelines.core/CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,20 @@ All notable changes to this package will be documented in this file.
...
 
 Version Updated
 The version number for this package has increased due to a version update of a related graphics package.
 
 ### Added
 - Debug Panels Framework See `IDebugDisplaySettingsQuery`.
+
+## [13.0.0] - 2021-09-01
+
+Version Updated
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.core/CHANGELOG.md:17]: removed this one, there is item here

**Review by ernestasKupciunas — APPROVED** [review]
I've done some smoke testing - imported all the URP required packages and checked if there are no conflicts/ errors. Works fine

**Pull request merged**